cluster/aws/util.sh

```diff
@@ -721,14 +723,11 @@ function create-iam-profile {
 
 # Creates the IAM roles (if they do not already exist)
 function ensure-iam-profiles {
-  aws iam get-instance-profile --instance-profile-name ${IAM_PROFILE_MASTER} || {
```

Comment:
@madhusudancs these IAM roles are created entirely by the `kube-up` script, so it's reasonable that they are destroyed by the script as well. This is the modus operandi for resource allocation- if we create it automatically, we clean it up automatically as well.